### ISSUE TITLE
More options to show price

### ISSUE THREAD

**Issue by imgidedin** [kalilistic/PriceCheck#29]
Would be nice to have the option to show it like this:

```
----------------------------------------------------
| Item       | Price (Normal)    | Price (HQ)
----------------------------------------------------
| Food X     | 1000 (Leviathan)  | 1600 (Famfrit)
| Tincture A | 700 (Behemoth)    | 1700 (Leviathan)
...
----------------------------------------------------
```

Just an example. Would make finding prices much easier. This is the reason I am not using the plugin currently, because I often need to know the difference between HQ and not.

**Comment by kalilistic** [comment]
Thanks I'll think about this one... my intention generally is to keep PriceCheck pretty simple. Marketboard already does a good job at more in-depth pricing.

**Comment by imgidedin** [comment]
Yeah, I have been using Marketboard, but I really wanted a plugin that would just tell me which server is the lowest NQ/HQ with a simple glance. Ah, I want to quickly sell this, which is the lowest in the datacenter? 600, awesome, 500 it is.

But that is me. You do what you think it is the best for the purpose of the plugin. Thanks for listening at least :)

**Comment by SgtWiking** [comment]
Same question.
Would be nice if there was an option to switch between "price check on my server"/"price check on my data center".

On universalis.app I always use the minimum data center price to adjust my prices, some stuff can vary by like 40-50% and people will server hop to buy the cheapest in that case.
Cheers
